guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -173,7 +173,7 @@ This approach allows for greater standardization but it leads to an increased nu
 The optimal solution depends on the nature of your host environment.
 Avoid creating a large number of content views, but keep in mind that the size of a content view affects the speed of related operations (publishing, promoting).
 Also make sure that when creating a subset of packages for the content view, all dependencies are included as well.
-Note that kickstart repositories should not be added to content views, as they are used for host provisioning only.
+Note that Kickstart repositories should not be added to content views, as they are used for host provisioning only.
```

Comment:
What a monster file! Aneta has been working on modularizing the planning guide, so probably no reason for me to file a bug.

Reply:
Yes :) Anything under topics/ needs to be reviewed and it's on my to-do list for the next few weeks.